chatcommands.go

```diff
@@ -24,7 +24,7 @@ type CommandFunction func(client *Client, args []string) (string, error)
 
 var commands = &CommandControl{
 	user: map[string]Command{
-		common.CNMe.String(): Command{
+		common.CNMe.String(): {
```

Comment:
Do not remove the explicit type here.  I know it'll work without it, but I want it there.  Same with the rest in this file.

Reply:
VsCode auto code cleaning... I'll manage that soon.